src/components/PortfolioPage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { MDXProvider } from '@mdx-js/react';
+import { MDXRenderer } from 'gatsby-plugin-mdx';
+import { Link } from 'gatsby';
+
+const shortcodes = { Link }; // Provide common components here
+
+export default function PortfolioPage({ data: { mdx } }) {
+  return (
+    <div>
+      <h1>{mdx.frontmatter.name}</h1>
+      <MDXProvider components={shortcodes}>
```

Comment:
What exactly does PortfolioPage do in conjunction with PortfolioModal? They look like they do the same thing, but not too sure how all of it works.

The code here looks very similar to the code in PortfolioModal.

Reply:
Nope, we'll have both a PortfolioModal and PortfolioPage

Reply:
Will handle in #58